sickbeard/providers/btdigg.py

```diff
@@ -0,0 +1,176 @@
+# Author: Jodi Jones <venom@gen-x.co.nz>
+# URL: http://code.google.com/p/sickbeard/
+#
+#Ported to sickrage by: matigonkas
+#
+# This file is part of Sick Beard.
+#
+# Sick Beard is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Sick Beard is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Sick Beard.  If not, see <http://www.gnu.org/licenses/>.
+
+import re
+import urllib, urllib2, cookielib
```

Comment:
don't need this with the other changes.
`import urllib, urllib2, cookielib`
becomes
`import requests`